errors.go

```diff
@@ -38,6 +38,7 @@ var (
 	ErrWithDistributedElectorNil     = fmt.Errorf("gocron: WithDistributedElector: elector must not be nil")
 	ErrWithDistributedLockerNil      = fmt.Errorf("gocron: WithDistributedLocker: locker must not be nil")
 	ErrWithDistributedJobLockerNil   = fmt.Errorf("gocron: WithDistributedJobLocker: locker must not be nil")
+	ErrWithWithIdentifierNil         = fmt.Errorf("gocron: WithIdentifier: identifier must not be nil")
```

Comment:
Let's remove the extra `with` 
```suggestion
	ErrWithIdentifierNil         = fmt.Errorf("gocron: WithIdentifier: identifier must not be nil")
```

Reply:
woops :sweat_smile:

Reply:
@JohnRoesler fixed